app/components/Weather.jsx

```diff
@@ -0,0 +1,58 @@
+import FiveDayWeather from "./FiveDayWeather";
+import { useDispatch } from "react-redux";
+import { setDefaultLocation } from "../store/slices/locations";
+import Graphs from "./Graphs";
+import Modal from "./Modal";
+
+
+export default function Weather ({ weatherDetails }) {
+  const dispatch = useDispatch();
+
+  return (
+      <>
+        <div className="d-flex justify-content-evenly mb-3">
+          <a className="no-decorations" data-bs-toggle="collapse" href={`#collapseFiveDay-${weatherDetails.id}`} role="button" aria-expanded="false" aria-controls={`collapseFiveDay-${weatherDetails.id}`}>
+            Five Day Forecast
+          </a>
+          <a className="no-decorations" data-bs-toggle="collapse" href={`#collapseGraphs-${weatherDetails.id}`} role="button" aria-expanded="false" aria-controls={`collapseGraphs-${weatherDetails.id}`}>
+            Five Day Graphs
+          </a>
+        </div>
+        <div className="accordion" id={`accordion-${weatherDetails.id}`}>
+          <div className="collapse" id={`collapseFiveDay-${weatherDetails.id}`} data-bs-parent={`#accordion-${weatherDetails.id}`}>
+            <div className="card card-body mb-3">
+              <FiveDayWeather fiveDayWeather={weatherDetails.fiveDayWeather} />
+            </div>
+          </div>
+          <div className="collapse" id={`collapseGraphs-${weatherDetails.id}`} data-bs-parent={`#accordion-${weatherDetails.id}`}>
+            <div className="card card-body mb-3">
+              <Graphs fiveDayWeather={weatherDetails.fiveDayWeather}/>
+            </div>
+          </div>
+        </div>
+      <div className="text-start mb-1">
```

Comment:
Would be great for this to show if the selected city was not already selected as default :)